src/Umbraco.Web.BackOffice/Controllers/BackOfficeController.cs

```diff
@@ -74,6 +84,84 @@ public async Task<IActionResult> Default()
                 () => View(viewPath));
         }
 
+        [HttpGet]
+        public async Task<IActionResult> VerifyInvite(string invite)
+        {
+            //if you are hitting VerifyInvite, you're already signed in as a different user, and the token is invalid
+            //you'll exit on one of the return RedirectToAction(nameof(Default)) but you're still logged in so you just get
+            //dumped at the default admin view with no detail
+            if (_webSecurity.IsAuthenticated())
+            {
+                await _signInManager.SignOutAsync();
+            }
+
+            if (invite == null)
+            {
+                _logger.Warn<BackOfficeController>("VerifyUser endpoint reached with invalid token: NULL");
+                return RedirectToAction(nameof(Default));
+            }
+
+            var parts = System.Net.WebUtility.UrlDecode(invite).Split('|');
```

Comment:
I can see that we have been using both `WebUtility.UrlEncode/UrlDecode` or `HttpUtility.UrlEncode/UrlDecode` throughout the code. Would be nice to just stick to, for example, `WebUtility.UrlEncode/UrlDecode` as there is a difference in the casing for the encoded characters. 

For Windows, it does not matter but in Linux they are different

Reply:
We cannot use HttpUtility, that is part of the System.Web namespace which is .net framework only.

Reply:
Think we can still use it in .net core - [link](https://docs.microsoft.com/en-us/dotnet/api/system.web.httputility.urlencode?view=netcore-3.1) -> for example we use it in **Umbraco.Web.BackOffice** (for some controllers or extensions) and in **Umbraco.Web.Website** (for action result) projects

Reply:
We must absolutely not use it in our .net core projects. This is problematic! I'm unsure how this has happened. All references to `using System.Web;` in our netcore/netstandard projects must not exist. @bergmania did you know about this? We still have references to `System.Web` in netcore/netstandard projects 😥

Reply:
I was not aware... From what I can see, it is luckily only `HttpUtility` that is used. I think the UrlEncode/UrlDecode can be replaced one to one, but we need to find an alternative to `HttpUtility.ParseQueryString`